projects/nokogiri/build.sh

```diff
@@ -57,10 +57,19 @@ then
   export SANITIZER_OPTS="-g -fprofile-instr-generate -fcoverage-mapping"
   export SANITIZER_LINK=""
 fi
+if [ "$SANITIZER" = "memory" ]
+then
+  export SANITIZER_OPTS="-fsanitize=memory -fPIE -pie -Wno-unused-command-line-argument"
+  export SANITIZER_LINK="$(find $($LLVM_CONFIG --libdir) -name libclang_rt.msan_cxx-x86_64.a | head -1)"
+fi
+if [ "$SANITIZER" = "introspector" ]
+then
+  export SANITIZER_OPTS="-fno-omit-frame-pointer -gline-tables-only -fuse-ld=gold -flto -Wno-unused-command-line-argument"
+  export SANITIZER_LINK=""
+fi
 
-
-export CXXFLAGS="-O0 $CXXFLAGS $SANITIZER_OPTS"
-export CFLAGS="-O0 $CFLAGS $SANITIZER_OPTS"
+export CXXFLAGS="-O0 -fno-inline-functions $CXXFLAGS $SANITIZER_OPTS"
+export CFLAGS="-O0 -fno-inline-functions  $CFLAGS $SANITIZER_OPTS"
```

Comment:
Am not sure I understand, what is the point of setting all these environment variables? These are set by the OSS-Fuzz infra itself, why is there a need to redo it?

Reply:
I'll give it a try without defining any of them myself, I just thought if I did them myself I didn't have to worry about external factors changing such as predefined environment variables, my bad

Reply:
alrighty made those changes @DavidKorczynski thanks for the suggestions! I'll make sure to not make that mistake in future integrations